t/plugin/key-auth.t

```diff
@@ -585,3 +585,84 @@ GET /hello?auth=auth-one
 auth: auth-one
 --- no_error_log
 [error]
+
+
+
+=== TEST 26: add consumer with plugin and username
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/consumers',
+                ngx.HTTP_PUT,
+                [[{
+                    "username": "bob",
+                    "plugins": {
+                        "key-auth": {
+                            "key": "12345:abc"
+                        }
+                    }
+                }]]
+                )
+
+            if code >= 300 then
+                ngx.status = code
+            end
+            ngx.say(body)
+        }
+    }
+--- request
+GET /t
+--- response_body
+passed
+--- no_error_log
+[error]
+
+
+
+=== TEST 27: add key-auth plugin with var_combination
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                ngx.HTTP_PUT,
+                [[{
+                    "plugins": {
+                        "key-auth": {
+                            "var_combination": "${http_client_id}:${http_client_secret}"
```

Comment:
The example in the document is "$remote_addr $consumer_name", and there is an extra colon in the test case. So which one is right?

Reply:
The documentation is just an example of what can be configured, it is not the same as the test case, and the documentation and the test case are not the same for the users. There is no right or wrong here. But I can unify them.

Reply:
This confuses me, how should I configure it?

Reply:
OK. I'll add some examples and test cases later.